sql/testdata/builtin_function

```diff
@@ -0,0 +1,78 @@
+query I colnames
+SELECT LENGTH('roach7')
+----
+LENGTH('roach7')
+6
+
+statement error LENGTH: argument type mismatch: string expected, but found int
+SELECT LENGTH(23)
+
+statement error LENGTH: incorrect number of arguments: 1 vs 2
+SELECT LENGTH('ab', 'df')
```

Comment:
in postgres LENGTH has a two-argument form:
```
length(string bytea, encoding name )
```

http://www.postgresql.org/docs/9.1/static/functions-string.html